scripts/update_vertexai_responses.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script replaces mock response files for Vertex AI unit tests with a fresh
+# clone of the shared repository of Vertex AI test data.
+
+RESPONSES_VERSION='v1.*' # The major version of mock responses to use
+REPO_NAME="vertexai-sdk-test-data"
+REPO_LINK="https://github.com/FirebaseExtended/$REPO_NAME.git"
+
+cd "$(dirname "$0")/../packages/vertexai/test-utils" || exit
+rm -rf "$REPO_NAME"
+git clone "$REPO_LINK" --quiet || exit
+cd "$REPO_NAME" || exit
+
+# Find and checkout latest tag matching major version
+TAG=$(git tag --sort=v:refname | grep "$RESPONSES_VERSION" | tail -n1)
```

Comment:
you don't need the `grep` command, you can use `git tag -l --sort=v:refname "$RESPONSES_VERSION"`